commons/src/connections/events.ts

```diff
@@ -66,9 +66,10 @@ export const fetchCalendars = async (
 export async function createEvent(
   event: Events.Event,
   query: Events.EventQuery,
+  component: "agenda" | "availability" = "agenda",
 ): Promise<Events.Event> {
   return fetch(
-    `${getMiddlewareApiUrl(query.component_id || "")}/agenda/events`,
+    `${getMiddlewareApiUrl(query.component_id || "")}/${component}/events`,
```

Comment:
The middleware scope for availability component is set as `/agenda/events` so we don't need to add availability here

Reply:
Removed. Thanks, Pooja!